ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/ktx/ThemeImageUrls.kt

```diff
@@ -4,8 +4,10 @@ package com.revenuecat.purchases.ui.revenuecatui.components.ktx
 
 import androidx.compose.foundation.isSystemInDarkTheme
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
 import com.revenuecat.purchases.paywalls.components.properties.ImageUrls
 import com.revenuecat.purchases.paywalls.components.properties.ThemeImageUrls
 
 internal val ThemeImageUrls.urlsForCurrentTheme: ImageUrls
-    @Composable get() = if (isSystemInDarkTheme()) dark ?: light else light
+    @ReadOnlyComposable @Composable
```

Comment:
Not important, but I think we can make some composables like this read only composables.